### PR TITLE
Model cloud 1

### PR DESCRIPTION
**Pull request by DaniDapena**
add the model cloud to download the weights

### CODE REVIEW

File: model_cloud/cloud_api.py [hunk 1]
```diff
@@ -0,0 +1,79 @@
+import json
```
Comment (tchaton) [inline model_cloud/cloud_api.py:1]: Let's move this under lightning_diffusion as `lightning_diffusion/model_cloud/cloud_api.py`

File: model_cloud/setup_tools.py [hunk 2]
```diff
@@ -0,0 +1,88 @@
+#!/usr/bin/env python
```
Comment (tchaton) [inline model_cloud/setup_tools.py:1]: This shouldn't be required.

File: model_cloud/utils.py [hunk 3]
```diff
@@ -0,0 +1,67 @@
+import json
```
Comment (tchaton) [inline model_cloud/utils.py:1]: Same as first comment.

File: dreambooth_component.py [hunk 4]
```diff
@@ -1,13 +1,18 @@
...
 from diffusers import StableDiffusionPipeline
-
+from model_cloud.cloud_api import download_from_lightning_cloud
+import zipfile
 
 class ServeDreamBoothDiffusion(BaseDiffusion):
 
     def setup(self):
+        download_from_lightning_cloud("daniela/stable_diffusion", version="latest",output_dir="model") 
+        with zipfile.ZipFile("model/checkpoint.zip", 'r') as zip_ref:
```
Comment (tchaton) [inline dreambooth_component.py:11]: Could we move this under the download call ?

File: lightning_diffusion/model_cloud/__init__.py [hunk 5]
```diff
@@ -0,0 +1,4 @@
+from .cloud_api import download_from_lightning_cloud
```
Comment (tchaton) [inline lightning_diffusion/model_cloud/__init__.py:1]: Better to use absolute path than relative ones.

**Review by tchaton — APPROVED** [review]
LGTM !

**Comment by Borda** [comment]
great work by @krshrimali!

**Pull request merged**